_docs/floodgate/forms.md

```diff
@@ -7,8 +7,8 @@ permalink: /floodgate/forms/
 # What is Cumulus?
 
 Bedrock Edition has a cool exclusive feature called Forms.<br>
-Cumulus is the new Forms API that we use in Geyser and Floodgate. The source code is available [here](https://github.com/GeyserMC/Cumulus).
-You can access the Cumulus API through the [Floodgate API](FloodgateApi).
+Cumulus is the new Forms API that we use in Geyser and Floodgate. The source code is available [here](https://wiki.geysermc.org/floodgate/forms/).
```

Comment:
We can probably just do `/floodgate/forms` for these?

Reply:
I'm not sure but should be possible yea.